modules/hcp-ibmz-adding-reg-ca-hostedcluster.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-adding-registry-ca-hostedcluster_{context}"]
+= Adding the Registry Certificate Authority to the Hosted Cluster
+
+When you are creating {hcp}, include the `additional-trust-bundle` and `image-content-sources`. This will allow the hosted cluster to inject the Certificate Authority into the Data plane workers so that the images are pulled from the registry.
+
+. Create `icsp.yaml` with image-content-sources. The `image-content-sources` information is available in the `ImageContentSourcePolicy` generated after mirrorring the images using `oc-mirror`.
++
+.Example ImageContentSourcePolicy file
+[source,terminal]
+----
+# cat icsp.yaml 
+- mirrors:
+  - <mirror_registry>/openshift/release
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+- mirrors:
+  - <mirror_registry>/openshift/release-images
+  source: quay.io/openshift-release-dev/ocp-release
+----
+. Provide the `additional-trust-bundle` certificate during the {hcp} creation to update the compute nodes with the certificates. Below is the sample {hcp} command to create the compute nodes with the certificates.
++
+[source,terminal]
+----
+$ hcp create cluster agent \
+    --name=<hosted_cluster_name> \ // <1>
+    --pull-secret=<path_to_pull_secret> \ // <2>
+    --agent-namespace=<hosted_control_plane_namespace> \ // <3>
+    --base-domain=<basedomain> \ // <4>
+    --api-server-address=api.<hosted_cluster_name>.<basedomain> \
+    --etcd-storage-class=<etcd_storage_class> \ // <5>
+    --ssh-key  <path_to_ssh_public_key> \ // <6>
+    --namespace <hosted_cluster_namespace> \ // <7>
+    --control-plane-availability-policy SingleReplica \
+    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp_release_image> \ // <8>
+    --<additional-trust-bundle> /var/lib/libvirt/registry/certs/registry.crt \ // <9>
+    --image-content-sources icsp.yaml
+----
+<1> Replace `<hosted_cluster_name>` with the name of your hosted cluster.
+<2> Replace the path to your pull secret, for example, `/user/name/pullsecret`.
+<3> Replace `<hosted_control_plane_namespace>` with the name of the hosted control plane namespace, for example, `clusters-hosted`.
+<4> Replace the name with your base domain, for example, `example.com`.
+<5> Replace the etcd storage class name, for example, `lvm-storageclass`.
+<6> Replace the path to your SSH public key. The default file path is `~/.ssh/id_rsa.pub`.
+<7> Replace `<hosted_cluster_namespace>` with your hosted cluster. Replace `<hosted_cluster_namespace>` with the name of your hosted cluster namespace.
```

Comment:
<hosted_cluster_namespace> 
here we are referring  <hosted_cluster_namespace> 2 times. 
we can give like this for <7>
Replace `<hosted_cluster_namespace>` with the name of your hosted cluster namespace.

Reply:
You are right, there was a repetition, i have removed <3> foot note.